### DISCUSSION TITLE
Xtreme-specific build

### DISCUSSION THREAD

**Discussion by ClaraCrazy** [akopachov/flipper-zero_authenticator#109] [General]
Just a question. Why are you only releasing for unleashed of all firmware? Since it is badKB from xtreme, why not actually do a build for that too?

**Comment by akopachov** [comment]
Well, for now I'm not releasing specifically for unleashed. Releases are based on "official stable" and "official dev" (which unleashed is compatible with) firmware. That "unleashed" in FAP file name is nothing more than just a keyword to simplify it for users to pick the right FAP.

Speaking of XTreme - I saw your excellent work on making some really nice API changes to let developers change MAC and bluetooth device name. And I'd like to use these features in Xtreme-specific build,

 So, just to be clear - I'm not ignoring it 😄. I just would like to make it good and use those unique capabilities which you guys providing, and it will take me some more time and effort to make sure that everything is working as expected.

Stay tuned 😉

**Comment by ClaraCrazy** [comment]
gotcha. join our discord and ping me mate. we can help :3

**Comment by ClaraCrazy** [comment]
inv is discord.gg/flipper-xtreme

**Reply by akopachov** [reply]
Joined. Thank you
